utils/index.js

```diff
@@ -1,12 +1,20 @@
 const importCwd = require('import-cwd');
 const semver = require('semver');
 
-const DEFAULT_VERSION = 2;
+const DEFAULT_VERSION = {
+  major: 2,
+  minor: 0,
+  patch: 0
```

Comment:
We'd better make the version that brought the last index change as the default VERSION because most of the people will come with the latest react-scripts version IMO especially when it is only a patch changes. Just update patch to `4`.